### PR TITLE
Create readme for future developers iterating on Swell

### PR DESCRIPTION
**Pull request by nattiechan**
`DEV_README.md` contains information that is more developer-specific.

Also, add the latest group members as contributors in package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -250,6 +250,26 @@
...
+    {
+      "name": "Jason Huang",
+      "email": "huangjj3@gmail.com",
+      "url": "https://https://github.com/jjhuang3"
+    },
+    {
+      "name": "Nattie Chan",
+      "email": "nattie.chan@gmail.com",
+      "url": "https://https://github.com/nattiechan"
+    },
```
Comment (nattiechan) [inline package.json:272]: @Fredkim21 @csuzukida @jjhuang3 Please double-check and make sure this is the email you want to include and the GitHub link is correct. TY :)

**Review by jjhuang3 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (nattiechan) [reply package.json]: done @jjhuang3

**Review by Fredkim21 — APPROVED** [review]
all correct for myself

File: package.json (hunk 1 above)
Comment (csuzukida) [reply package.json]: Could you change mine to chris.suzukida@gmail.com? Other than that, it looks good thanks

Comment (nattiechan) [reply package.json]: done @csuzukida

**Pull request merged**